src/metadata/entity_descriptor.rs

```diff
@@ -18,29 +18,29 @@ pub struct EntityDescriptor {
     pub entity_id: Option<String>,
     #[serde(rename = "ID")]
     pub id: Option<String>,
-    #[serde(rename = "ds:Signature")]
+    #[serde(rename = "Signature")]
```

Comment:
The prefixes were added when I say experimenting with serde serialization, but that didn't pan out. Out of curiosity, are you removing these because they caused issues for you, or just because they aren't needed for deserialization?

Reply:
I believe it actually didn't deserialize correctly